otx/algorithms/segmentation/adapters/mmseg/models/segmentors/mean_teacher_segmentor.py

```diff
@@ -21,19 +23,42 @@ class MeanTeacherSegmentor(BaseSegmentor):
     It creates two models and ema from one to the other for consistency loss.
     """
 
-    def __init__(self, orig_type=None, unsup_weight=0.1, semisl_start_iter=30, **kwargs):
+    def __init__(
+        self,
+        orig_type=None,
+        unsup_weight=0.1,
+        proto_weight=0.7,
+        aux_weight=0.1,
+        drop_percent=80,
+        num_iters_per_epoch=6000,
```

Comment:
I see. But I think it's better to set `None` as default value or remove default value because it can prevent using it without setting `num_iters_per_epoch` later.